### ISSUE TITLE
Orientation for customized number spinner

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#130]
For optimal accessibility and cross-platform usability, the number spinner's in this sim are implemented as sliders with the `aria-roledescription="number spinner"`

The number spinners are visually vertical, and this should be explicitly set on the slider to indicate that the Up Down Arrow keys are the keys of choice.

Interestingly, on May 10th our VoiceOver consultant indicated some confusion about what keys to use when she encountered the "number spinners". 

To the consultant the control seemed to be a horizontal slider working with the Left/Right Arrow keys, but the role was announced as "number spinner". Though the control is fully operational with all 4 arrow keys, in the consultant's mental model of a number spinner, she would typically only the Up and Down arrow keys. She hesitated wondering if the Up/Down Arrows would in fact work.

I think an explicit `orientation="vertical"` attribute on the vertical number spinner might help.

**Comment by zepumph** [comment]
Makes sense to me! I made the number spinner default to vertical, since I think that that is what most NumberSpinner and NumberPickers look like at phet. The mass controls are now vertical in orientation. Closing
